packages/samples/headless-ssr-commerce/app/_components/recommendation-list.tsx

```diff
@@ -1,29 +1,16 @@
-import {
-  Product,
-  Recommendations as RecommendationsController,
-  RecommendationsState,
-} from '@coveo/headless/ssr-commerce';
-import {useRouter} from 'next/navigation';
-import {useEffect, useState, FunctionComponent} from 'react';
+'use client';
 
-interface RecommendationsProps {
-  staticState: RecommendationsState;
-  controller?: RecommendationsController;
-}
+import {Product} from '@coveo/headless-react/ssr-commerce';
+import {useRouter} from 'next/navigation';
+import {FunctionComponent} from 'react';
+import {usePopularBoughtRecs} from '../_lib/commerce-engine';
 
-export const Recommendations: FunctionComponent<RecommendationsProps> = ({
-  staticState,
-  controller,
-}) => {
-  const [state, setState] = useState(staticState);
+export const Recommendations: FunctionComponent = () => {
```

Comment:
Do you mind changing this to keep the same style across all of our react components.

```suggestion
export default function Recommendations() {

```